docs/_docs/monitoring-metrics/system-views.adoc

```diff
@@ -337,23 +366,6 @@ This view exposes information about currently running scan queries.
 |===
 
 
-== SQL_QUERIES_HISTORY
```

Comment:
Did we remove this view or replaced it? Or did you remove this accidentally?

Reply:
This view described twice on this page.
Please, see line 415.